ci/gpu/build.sh

```diff
@@ -118,3 +118,8 @@ else
         py.test --cache-clear -vs `python -c "import distributed.diagnostics.tests.test_nvml as m;print(m.__file__)"`
     fi
 fi
+
+if [ -n "\${CODECOV_TOKEN}" ]; then
+    codecov -t \$CODECOV_TOKEN
```

Comment:
Is this really correct? My understand is that `\$` will escape the `$`, rather than get the variable's value. Is that really what's wanted or am I missing something?

Reply:
We might also consider skipping the environment variable altogether. It seems `codecov` [checks this environment variable by default]( https://github.com/codecov/codecov-python/blob/713d94113e9b1189f227adc6131f04c65e4f967b/codecov/__init__.py#L327 ).

Reply:
Ah, you're correct @pentschev , this was copy pasted from a script in gpuCI that gets injected, so we were escaping characters to handle the injection.

The upload token is a secret text inside Jenkins, Jenkins will always hide it, no need to worry about that.